video/filter/vf_d3d11vpp.c

```diff
@@ -72,6 +86,7 @@ struct priv {
     struct mp_image_pool *pool;
 
     struct mp_refqueue *queue;
+    int sr_state; // -1 for failed, 0 for off, 1 for on
```

Comment:
This can be enum, tho avoid awkward comments explaining what each value mean.

Reply:
updated